test/gtest/gtest_activation.cpp

```diff
@@ -0,0 +1,344 @@
+                                             dx);
+
+    if(fwdStatus == miopenStatusSuccess && bwdStatus == miopenStatusSuccess)
+        return miopenStatusSuccess;
+    else if(fwdStatus != miopenStatusSuccess)
+        return fwdStatus;
+    else
+        return bwdStatus;
+}
+
+TEST_P(TestActivation, ActivationFwdTest)
+{
+    const float alpha = 1.0f, beta = 0;
+    isBwdActivation = false;
+
+    miopenStatus_t status = miopenStatusUnknownError;
+
+    if(data_type == miopenFloat)
+    {
+        status = RunFwdActivation(get_handle(),
+                                  activ_desc,
+                                  &alpha,
+                                  std::get<0>(input).desc, // x
+                                  in_dev.get(),
+                                  &beta,
+                                  std::get<0>(output_gpu).desc, // y
+                                  out_dev.get());
+    }
+
+    EXPECT_EQ(status, miopenStatusSuccess) << "Forward activation failed. Config: " << activ_config;
+}
+
+TEST_P(TestActivation, ActivationBwdTest)
+{
+    const float alpha = 1.0f, beta = 0;
+    isBwdActivation = true;
+
+    miopenStatus_t status = miopenStatusUnknownError;
+
+    if(data_type == miopenFloat)
+    {
+        status = RunFwdBwdActivation(get_handle(),
+                                     activ_desc,
+                                     &alpha,
+                                     std::get<0>(input).desc, // x
+                                     in_dev.get(),
+                                     &beta,
+                                     std::get<0>(output_gpu).desc, // y
+                                     out_dev.get(),
+                                     std::get<0>(doutput).desc, // dy
+                                     dout_dev.get(),
+                                     std::get<0>(dinput_gpu).desc, // dx
+                                     din_dev.get());
+    }
+
+    EXPECT_EQ(status, miopenStatusSuccess)
+        << "Backward activation failed. Config: " << activ_config;
+}
+
+INSTANTIATE_TEST_SUITE_P(ActivationTestSuite,
+                         TestActivation,
+                         ::testing::Combine(::testing::Values(miopenFloat),
+                                            // miopenDouble),
+
+                                            ::testing::Values(miopenActivationPASTHRU,
+                                                              miopenActivationLOGISTIC,
+                                                              miopenActivationTANH,
+                                                              miopenActivationRELU,
+                                                              miopenActivationSOFTRELU,
+                                                              miopenActivationABS,
+                                                              miopenActivationPOWER,
+                                                              // miopenActivationCLIPPEDRELU,
```

Comment:
This is enabled in the original test.

Reply:
@atamazov , I believe the rational is to migrate to GoogleTest frame work once more test cases are covered. That's why these activation modes are there.

Reply:
The full Q is: Why miopenActivationCLIPPEDRELU is disabled here while it is enabled in the original test?

Reply:
Let me re-run this test with miopenActivationCLIPPEDRELU enabled.